spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1106,8 +1106,24 @@ For the conversion of inbound messages, especially when the target is a POJO, th
 
 Besides the conversions that it supports out of the box, Spring Cloud Stream also supports registering your own message conversion implementations.
 This allows you to send and receive data in a variety of custom formats, including binary, and associate them with specific `contentTypes`.
-In order to do so, you can create a class that extends `AbstractMessageConverter`
+In order to do so, you can create a class of type `org.springframework.messaging.converter.MessageConverter` and make it available in the application context.
```

Comment:
ok, I think this also means we need to use `AbstractMessageConverter` to get the list of custom message converters instead of just looking for the type `MessageConverter` list in the application context as [here] (https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java#L91)